feature-libs/product-configurator/rulebased/root/http-interceptors/configurator-bad-request.handler.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpErrorResponse, HttpRequest } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import {
+  ErrorModel,
+  GlobalMessageService,
+  GlobalMessageType,
+  HttpErrorHandler,
+  HttpResponseStatus,
+  Priority,
+} from '@spartacus/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ConfiguratorBadRequestHandler extends HttpErrorHandler {
+  responseStatus = HttpResponseStatus.BAD_REQUEST;
+
+  constructor(protected globalMessageService: GlobalMessageService) {
+    super(globalMessageService);
+  }
+
+  getPriority(): Priority {
+    return Priority.HIGH;
```

Comment:
Why does the priority need to be high?

Reply:
I will change to `Priority.NORMAL`.
I retested it and it worked as expected.
The reason, why the priority was set to high as far as I remember, war because of the 2 error messages in the case of the the quick-order.